docs/shrc_diag_util.md

```diff
@@ -120,7 +127,7 @@ Each entry in the output starts with a VM ID, so you can see which VM wrote the
 -Xshareclasses:printStats=<data_type1>[+<data_type2>][...],name=<cache_name>
 ```
 
-Displays summary information about the cache. For layered caches, some information is shown for the top layer cache only, and some is shown for all layers combined. To see information for the top layer cache only, use [`printTopLayerStats`](#printtoplayerstats).
+Displays summary information about the cache. For layered caches, `-Xshareclasses:printStats` shows some information for the top layer cache, and summary information (bytes and counts only) for all layers combined. To see information for the top layer cache only, use [`printTopLayerStats`](#printtoplayerstats).
 
 You can request more detail about items of a specific data type that are stored in the shared cache by using `printStats=<data_type>`. Use the plus symbol (+) to separate the data types. For example, use `printStats=romclass+url,name=myCache` to see information about `ROMClass` and `URL` items in all the layer caches of the cache called `myCache`. The valid data types are as follows (case insensitive):
```

Comment:
> of the cache called `myCache`

In the example bellow, the cache name is 'Cache1'

Reply:
Good catch. Updated.